src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -405,6 +419,10 @@ class MediaBaseViewer extends BaseViewer {
      * @return {void}
      */
     setTimeCode() {
+        if (!this.mediaControls) {
```

Comment:
Maybe could have a helper function for `hasMediaControls` or instantiate the `this.mediaControls` with an object in the interim until `loadUi` has been called?

Reply:
I attempted to move `loadUI` around instead so that the mediaEl callbacks can depend on mediaControls existing